backend/apps/pr_review_bot/models.py

```
@@ -0,0 +1,181 @@
+    # Suggestion
+    suggestion = models.TextField(blank=True)
+    suggested_code = models.TextField(blank=True)
+    
+    # Status
+    is_fixed = models.BooleanField(default=False)
+    fixed_at = models.DateTimeField(null=True, blank=True)
+    
+    created_at = models.DateTimeField(auto_now_add=True)
+    
+    class Meta:
+        ordering = ['-severity']
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)